### ISSUE TITLE
Is it possible to change word boundaries for `HopWord`?

### ISSUE THREAD

**Issue by Tylerc230** [phaazon/hop.nvim#247]
I've noticed that hop doesn't use the same definition of word that neovim uses. Vim (when tapping `w` for instance) uses whitespace to delimit words but hop uses alphanumerics (I think). This means that '(' and other non alphanumeric characters aren't highlighted. For instance in the following code, I'd like the '(' at the end of the line to be highlighted for hopping to  when I `HopWord` but the final 'F' is that last thing to get highlighted. Is this configurable?
```
const DeleteReactionConfirmationView: React.FC = () => {
```

**Comment by phaazon** [comment]
Yes, it uses `\k` by default. You can easily create your own by having a look at the API:

```
:h hop.jump_target.regex_by_searching
```
